_dashboards/admin-ui-index/rollover.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Rollover management
+parent: Index and snapshot management in OpenSearch Dashboards
+nav_order: 40
+---
+
+# Rollover management
+Introduced 2.6
+{: .label .label-purple }
+
+OpenSearch Dashboards allows you to perform an [index rollover]({{site.url}}{{site.baseurl}}/im-plugin/ism/error-prevention/index/#rollover) operation with **Index Management**.
+
+## Data Steams
+
+To perform a rollover operation on data streams, perform the following steps:
+
+1. Under **Index Management**, choose **Data streams**.
+
+2. Choose **Actions**, then choose **Rollover** as seen in the following image.
+
+    ![Rollover]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover1.png)
+
+3. Under **Configure source**, select the source data stream you would like to preform the rollover operation.
+
+4. Choose **Rollover** as seen in the following image.
+
+    ![Rollover]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover3.png)
+
+## Aliases
+
+To perform a rollover operation on aliases, perform the following steps:
+
+1. Under **Index Management**, choose **Aliases**.
+
+2. Select **Actions**, then choose **Rollover** as seen in the following image.
+
+    ![Rollover]({{site.url}}{{site.baseurl}}/images/admin-ui-index/rollover2.png)
+
+3. Under **Configure source**, select the source data stream you would like to preform the rollover operation.
```

Comment:
The "data stream" should be alias.
Further more, if the alias does not have a writing index, user should choose one before doing rollover.

![image](https://user-images.githubusercontent.com/13493605/219998203-b111ae6d-9ffd-475f-8b39-5b457b04fde2.png)

Reply:
When rolling over an alias, there is one more section used to define the new writing index.
![image](https://user-images.githubusercontent.com/13493605/219998519-89ff3562-0458-43fa-8dd8-10d17f0b32a9.png)